steward/src/allocation.rs

```diff
@@ -52,7 +59,7 @@ pub async fn get_connections() -> Result<Connections, Error> {
     Ok(Connections { grpc, contact })
 }
 
-pub async fn decide_rebalance(tick_range: Vec<somm::TickRange>, cellar_address: H160) {
+pub async fn decide_rebalance(tick_range: Vec<somm::TickRange>, cellar_address: H160, pair_id: String) {
```

Comment:
Let's rename `pair_id` to `cellar_id` here to reflect the different argument.

Reply:
So what @cbrit really wants is for me to totally remove pair_id from the decide_rebalance function and replace pair_id with cellar_address in line 31, 71, 105, 154

Reply:
Yes, elsewhere when you are calling `decide_rebalance`, this new third argument is the `cellar_id` but it's still named `pair_id` here. Since we're replacing `pair_id` with `cellar_id` throughout the allocation code, let's rename the argument to `cellar_id` as well.

Reply:
fixed with this commit: https://github.com/PeggyJV/steward/pull/57/commits/944375d8703aeb9ab07c8c026461c33da3424dcf